enterprise/server/backends/distributed/distributed.go

```diff
@@ -382,18 +381,19 @@ func (c *Cache) addLookasideEntry(r *rspb.ResourceName, data []byte) {
 	c.log.Debugf("Set %q in lookaside cache", k)
 }
 
-func (c *Cache) getLookasideEntry(r *rspb.ResourceName) ([]byte, error) {
+// getLookasideEntry returns the resource and if it was found in the lookaside
+// cache.
+func (c *Cache) getLookasideEntry(r *rspb.ResourceName) (data []byte, found bool) {
 	if !c.lookasideCacheEnabled() {
-		return nil, status.NotFoundError("lookaside cache disabled")
+		return nil, false
 	}
-	k, err := lookasideKey(r)
-	if err != nil {
-		c.log.Debugf("Not getting lookaside entry: %s", err)
-		return nil, err
+	k, ok := lookasideKey(r)
+	if !ok {
+		c.log.Debugf("Not getting lookaside entry for resource: %s", r)
+		return nil, false
 	}
 
 	c.lookasideMu.Lock()
-	found := false
```

Comment:
Could we bring this back and switch back to the non-named return values? They have been a pretty error-prone pattern for us in the past so we prefer not to use them